docs/authorization.md

```diff
@@ -230,6 +230,41 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+## Consumer Permission Claims Authorizer
+
+The requested resource type might be a claimed resource of a PermissionClaim in an APIExport
+referenced by an APIBinding in the workspace being accessed.
+In this case access to the requested resource is constrained by the field `consumerVerbs` of the underlying permission claim.
+
+If the requested resource type is claimed by different API Exports in the same workspace,
+the intersection of all verbs of all APIExports is being granted to the user.
+
+Valid values for the `consumerVerbs` field follow the same rules as the `verbs` field in PolicyRules:
+
+- It is required.
```

Comment:
I tend to
1. default consumerVerbs with "*" 
2. default providerVerbs with empty list. Empty list is not good though as we should not distinguish between unset and empty (broken yaml/json libraries and all that). So maybe we can use `["none"|` for that purpose.

Reply:
As discussed:

- no default: forces to make a conscious decision
- make verbs required
- avoid empty list similar to what kube does as the semantics are not clear